insights/client/constants.py

```diff
@@ -87,3 +87,4 @@ class InsightsConstants(object):
     rhsm_facts_file = os.path.join(os.sep, 'etc', 'rhsm', 'facts', 'insights-client.facts')
     # In MB
     archive_filesize_max = 100
+    ip_obfuscation_blacklist = ("insights.specs.Specs.installed_rpms",)
```

Comment:
Can we call it a "deny list" or an "exclude list" instead of a "black list"?

Reply:
“deny list” doesn’t sound correct to me as we do not deny access or similar. “exclude list” is much better. I was thinking about an “exempt list”, too.

Reply:
Yea, I think either of those would work.